client/asset/dcr/spv.go

```diff
@@ -773,7 +774,12 @@ func (w *spvWallet) SyncStatus(ctx context.Context) (bool, float32, error) {
 		return false, 0, nil
 	}
 
-	return bestHeight == height, float32(height) / float32(bestHeight), nil
+	synced, progress := w.spv.Synced(), float32(height)/float32(bestHeight)
+	if progress == 1 && !synced {
```

Comment:
Sorry to nit, but to avoid any backward progress, even if small, I think this should be

`if progress > 0.99 & !synced {`

Also that's 99%, so shouldn't we use 0.999?